backend/src/main/java/utils/WordPhoneticGetter.java

```diff
@@ -0,0 +1,79 @@
+package com.animalrhymes.server;
+
+import com.jayway.jsonpath.*;
+import java.util.*;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/*
+ * Class with one sole purpose: return the phonetic of some word using utility method getPhonetic(String word)
+ */
+public class WordPhoneticGetter {
+    private static final String USER_AGENT = "Mozilla/5.0";
+    private static final String GET_URL = "https://od-api.oxforddictionaries.com:443/api/v2/entries/en-us/";
+	private static final String PRONUNCIATION_FIELD = "fields=pronunciations";
+	private static int current = 0;
+
+    public static String getPhonetic(String word) {
+    	// API only handles 500 rpm. This is a lazy way to handle this.
+    	current += 1;
+    	if (current == 499) {
+			current = 0;
+			try {
+				Thread.sleep(60000);
+			} catch(InterruptedException ex) {
+				Thread.currentThread().interrupt();
+			}
+		}
+
+        try {
+            String json = sendGET(word);
+            List<LinkedHashMap> phonetic = JsonPath.read(json, "$.results[0].lexicalEntries[0].pronunciations");
+
+            for (LinkedHashMap lhm : phonetic) {
+            	if (lhm.get("phoneticNotation") != null && lhm.get("phoneticNotation").equals("IPA")) {
+            		return (String) lhm.get("phoneticSpelling");
+				}
+			}
+
+            return "";
+        } catch (IOException | PathNotFoundException ex) {
+            System.err.println("IOException thrown!");
+            ex.printStackTrace();
+            return "";
```

Comment:
Have you considered returning a 500 to the client instead of failing silently? It could make the errors more visible to the client and easier to debug.

Reply:
Had not. In a future PR I will take this into consideration. For MVP I think it's fine right now.